src/python/pants/backend/scala/compile/scalac_plugins.py

```diff
@@ -86,9 +86,8 @@ class GlobalScalacPlugins:
     classpath: MaterializedClasspath
 
     def args(self, prefix: str | None = None) -> Iterator[str]:
-        scalac_plugins_arg = ":".join(self.classpath.classpath_entries(prefix))
```

Comment:
All the documentation I find is either for `-Xplugin:<jar path>` (old) or `-Xplugin <path1>,...`. I found no documentation this accepts classpath format. The description of the PR is confusing if my research was correct, in which case the syntax we were using was simply bogus.

Reply:
Good call: it looks like comma separated works here as well... so that seems like the root cause. Will leave as is and fix the description.